### PR TITLE
makefile: add top_srcdir into CPPFLAGS for yocto

### PR DESCRIPTION
**Pull request by wudping**
to integrate libyami into yocto, we should add top_srcdir into cppflags
to make it compiled well.

Signed-off-by: wudping <dongpingx.wu@intel.com>

### CODE REVIEW

File: examples/Makefile.am [hunk 1]
```diff
@@ -17,6 +17,7 @@ AM_CFLAGS = \
 	-I../tests \
```
Comment (chivakker) [inline examples/Makefile.am]: Can you do something like this:

extra_Includes = \
-I $(top_srcdir) \ #to compile within yocto

and then you add the extra_includes wherever you need.  

That way can help us identify in the future why this was added and maybe modify it.

Comment (wudping) [inline examples/Makefile.am]: okay, thanks @chivakker

File: examples/Makefile.am [hunk 2]
```diff
@@ -73,21 +74,24 @@ VPP_INPUT_LIBS = \
...
 VPP_INPUT_CFLAGS = \
 	$(LIBVA_CFLAGS) \
+	-I$(top_srcdir) \
 	$(NULL)
 
 if ENABLE_X11
 simpleplayer_LDADD = $(YAMI_DECODE_LIBS)
 simpleplayer_LDFLAGS = $(LIBYAMI_CFLAGS)
 simpleplayer_SOURCES = simpleplayer.cpp $(DECODE_INPUT_SOURCES)
+simpleplayer_CPPFLAGS = $(LIBYAMI_CFLAGS) $(LIBDRM_CFLAGS) $(AM_CPPFLAGS) -I$(top_srcdir)
```
Comment (xuguangxin) [inline examples/Makefile.am]: Why we need this?We already added them to AM_CFLAGS/AM_CPPFLAGS

File: examples/Makefile.am [hunk 3]
```diff
@@ -73,21 +74,24 @@ VPP_INPUT_LIBS = \
 
 VPP_INPUT_CFLAGS = \
 	$(LIBVA_CFLAGS) \
+	-I$(top_srcdir) \
```
Comment (xuguangxin) [inline examples/Makefile.am]: seems no place will use VPP_INPUT_CFLAGS , why we need add this

File: examples/Makefile.am (hunk 2 above)
Comment (wudping) [inline examples/Makefile.am]: thanks @xuguangxin , I will modify it.

File: examples/Makefile.am (hunk 3 above)
Comment (wudping) [inline examples/Makefile.am]: thanks @xuguangxin , I will modify it.

**Pull request merged**